src/components/login-button.tsx

```diff
@@ -2,16 +2,19 @@ import { Link } from 'react-router-dom';
 import { AppRoute, AuthorizationStatus } from '../const';
 import { useAppDispatch, useAppSelector } from '../hooks';
 import { logoutAction } from '../store/api-actions';
+import { getAuthorizationStatus, getAvatarUrl } from '../store/user-process/selectors';
+import { cleanFavoriteFilms } from '../store/app-data/app-data';
 
 export function LoginButton (): JSX.Element {
   const dispatch = useAppDispatch();
 
   const onSignOut = () => {
     dispatch(logoutAction());
+    dispatch(cleanFavoriteFilms);
```

Comment:
```suggestion
    dispatch(cleanFavoriteFilms());
```
Разве TS тут не ругался?